api_tests/btsieve/config.toml

```diff
@@ -0,0 +1,6 @@
+ledgers = ["bitcoin", "ethereum"]
```

Comment:
can you `git mv` this file instead?

Reply:
As discussed, I tried but the problem is that both the name of the folder and the contents of the file changed, and `git` seems to only be able to represent this by deleting and adding, rather than by modifying.